generics/constants/common.js

```diff
@@ -69,5 +69,13 @@ module.exports = {
     "CREATE_SURVEY": "/create-survey/",
     "SUBMISSION_STATUS_COMPLETED" : "completed",
     "EXPIRED": "expired",
-    "STARTED":"started"
+    "STARTED":"started",
+    "AWS_SERVICE": "AWS",
+    "NO_OF_MINUTES": 30,
+    "NO_OF_EXPIRY_TIME": 60,
```

Comment:
@Prajwal17Tunerlabs We need to add two more env Variables
PRESIGNED_URL_EXPIRY_IN_SECONDS (in seconds, when we create signed url , by default pass it )
DOWNLOADABLE_URL_EXPIRY_IN_SECONDS (in seconds, when ever downloadable url created for private bucket pass this value as expiry )

update envVariables.js
sample : 
DOWNLOAD_URL_EXPIRATION_DURATION: {
		message: 'Required downloadable url expiration time',
		optional: true,
		default: 300,
	},

Reply:
have added the specified env variables in envVaribales.js and .env files